_search-plugins/cross-cluster-search.md

```diff
@@ -297,3 +303,56 @@ curl -k -XPUT -H 'Content-Type: application/json' -u 'admin:<custom-admin-passwo
   }
 }'
 ```
+
+## Proxy settings
+You can configure cross-cluster search on a cluster running behind a proxy. There are many ways to configure a reverse proxy and various proxies to choose from. The following example demonstrates the basic NGINX reverse proxy configuration without TLS termination, though there are many proxies and reverse proxies to choose from.  For this example to work, OpenSearch is expected to be running with both transport and HTTP TLS encryption enabled. For more information about configuring TLS encryption, see [Configuring TLS certificates]({{site.url}}{{site.baseurl}}/security/configuration/tls/).
+
+### Proxy configuration
+The basic NGINX configuration for HTTP and transport communication follows:
+
+```
+stream {
+    upstream opensearch-transport {
+        server <opensearch>:9300;
+    }
+    upstream opensearch-http {
+        server <opensearch>:9200;
+    }
+    server {
+        listen 8300;
+        ssl_certificate /.../opensearch-<VERSION>/config/esnode.pem;
+        ssl_certificate_key /.../opensearch-<VERSION>/config/esnode-key.pem;
+        ssl_trusted_certificate /.../opensearch-<VERSION>/config/root-ca.pem;
+        proxy_pass opensearch-transport;
+        ssl_preread on;
+    }
+    server {
+        listen 443;
+        listen [::]:443;
+        ssl_certificate /.../opensearch-<VERSION>/config/esnode.pem;
+        ssl_certificate_key /.../opensearch-<VERSION>/config/esnode-key.pem;
+        ssl_trusted_certificate /.../opensearch-<VERSION>/config/root-ca.pem;
+        proxy_pass opensearch-http;
+        ssl_preread on;
+    }
+}
+```
+The listening ports for HTTP and transport communication are set to `443` and `8300` respectively. 
+
+### OpenSearch configuration
+The remote cluster can be configured to point to the `proxy` using the following command:
+
+```bash
+curl -k -XPUT -H 'Content-Type: application/json' -u 'admin:<custom-admin-password>' 'https://opensearch:9200/_cluster/settings' -d '
+{
+  "persistent": {
+    "cluster.remote": {
+      "opensearch-remote-cluster": {
+        "mode": "proxy",
+        "proxy_address": "<remote-cluster-proxy>:8300"
+      }
+    }
+  }
+}'
+```
+Note the previously defined port `8300` configured in the [Proxy configuration]({{site.url}}{{site.baseurl}}/search-plugins/cross-cluster-search/#proxy-configuration) section.
```

Comment:
Do we mean "Note that port `8300` was previously configured in the..."?

Reply:
@natebower thats been updated now